lib/storage/bucket.js

```diff
@@ -218,14 +219,16 @@ function Bucket(storage, name) {
  * @param {function=} callback - The callback function.
  *
  * @example
- * var 2013logs = bucket.file('2013-logs.txt');
- * var 2014logs = bucket.file('2014-logs.txt');
```

Comment:
While we're requesting features for the JS tokenizer, I'd like to request supporting hyphens.